### ISSUE TITLE
Is there an env for TLS?

### ISSUE THREAD

**Issue by Gapoly** [bpatrik/pigallery2#957]
I'm looking for the feature to add a TLS certificate the web interface. Is there a environment variables to do that or not?

**Comment by bpatrik** [comment]
no, please use a reverse proxy in front of the app, like nginx. See docker docs for that.
